policyengine_us/parameters/gov/states/nm/tax/income/credits/cdcc/income_limit.yaml

```diff
@@ -0,0 +1,11 @@
+description: New Mexico provides a child day care credit for filing units with modified gross income at or below this limit.
+metadata:
+  unit: currency-USD
+  label: New Mexico child day care credit modified gross income limit
+  reference:
+    - title: 2021 New Mexico Income Tax Form Instructions, Page RC-7
+      href: https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/608063af-0c4c-4f28-ba42-cf40e04557ca/PITbook2021.pdf#page=63
+    - title: 2022 New Mexico Income Tax Form Instructions, Page RC-7
+      href: https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/1afc56af-ea90-4d48-82e5-1f9aeb43255a/PITbook2022.pdf#page=72
+values:
+  2021-01-01: 30_160
```

Comment:
This is 40*52*7.25*2 so should give the same result

Reply:
Yes.  Original minimum wage derivation of income limit has been restored.